actionlist/app/containers/ActionListContainer.js

```diff
@@ -1,17 +1,69 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   View,
+  DrawerLayoutAndroid,
 } from 'react-native';
 
 import Header from '../components/header';
 import DisplayList from '../components/display_list';
+import SideMenu from '../components/SideMenu';
+import { toggleDrawer } from '../actions/action_items';
+
+class ActionListContainer extends Component {
```

Comment:
I asked this question in stackoverflow and then realized since the drawer is open and close uses ref, I had to have a state variable for the component which gets toggled when it is triggered by an action.